CMakeLists.txt

```diff
@@ -38,15 +45,15 @@ if(NOT CHRONOLOG_INSTALL_INCLUDE_DIR)
     set(CHRONOLOG_INSTALL_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include)
 endif()
 if(NOT CHRONOLOG_INSTALL_DATA_DIR)
```

Comment:
Do you think it's a good idea to rename this to `CHRONOLOG_INSTALL_CONF_DIR` and leave `CHRONOLOG_INSTALL_DATA_DIR` for potential actual data in the future? If we follow the same pattern, we can also have `CHRONOLOG_INSTALL_BIN_DIR` and `CHRONOLOG_INSTALL_LIB_DIR` if we want.

Reply:
(1) I would check that CMAKE_BUILD_TYPE specified as Debug or Release first  and exit if anything other than these 2 is supplied because we don't support anything else
(2) define global CHRONOLOG_INSTAL_PREFIX  as ${CMAKE_INSTALL_PREFIX}/${CMAKE_BUILD_TYPE} )
or just redefine CMAKE_INSTALL_PREFIX as ${CMAKE_INSTALL_PREFIX}/${CMAKE_BUILD_TYPE} )